src/Controls/src/Core/Compatibility/Handlers/Shell/Android/ShellFlyoutTemplatedContentRenderer.cs

```diff
@@ -534,6 +529,18 @@ void UpdateElevation()
 					ElevationHelper.SetElevation(view, View);
 			}
 
+			protected override void OnMeasure(int widthMeasureSpec, int heightMeasureSpec)
+			{
+				var width = widthMeasureSpec.GetSize();
```

Comment:
Is this needed? Doesn't seem to be used and also this appears to be an "empty" override? Debugging fun?